### PR TITLE
ADD missing changes for vm-manager

### PR DESCRIPTION
**Pull request by bkothapx**
Added missing changes from vertical S

### CODE REVIEW

**Review by prashanth-intel — APPROVED** [review]
LGTM.

**Review by bnkoppad — APPROVED** [review]

File: bsp_diff/caas/device/intel/civ/host/vm-manager/0021-Revert-Add-config-ini-as-part-of-package.patch [hunk 1]
```diff
@@ -0,0 +1,84 @@
+From a3975e4d8eccd1792b472d8c3823e605872cf220 Mon Sep 17 00:00:00 2001
+From: "Kothapeta, BikshapathiX" <bikshapathix.kothapeta@intel.com>
+Date: Mon, 10 Apr 2023 20:04:19 +0530
+Subject: [PATCH 3/7] Revert "Add config ini as part of package."
```
Comment (basilchew) [inline bsp_diff/caas/device/intel/civ/host/vm-manager/0021-Revert-Add-config-ini-as-part-of-package.patch:4]: This seems to be reverting the vertical patch 0005-Add-config-ini-as-part-of-package.patch.
And there is another patch 0023-Add-config-ini-as-part-of-package.patch to add it back.
Please check.

File: bsp_diff/caas/device/intel/civ/host/vm-manager/0022-Inhibit-the-Host-suspend-activity.patch [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+From: Shwetha B <shwetha.b@intel.com>
+Date: Fri, 9 Jul 2021 11:41:52 +0530
+Subject: [PATCH 4/7] Inhibit the Host suspend activity
+
+This patch sets the sleep-inhibitor in the Host
+so that when the sleep-inhibitor service is started
+it inhibits the Host suspend activity from any user.
+This will stop host suspend only when the Android screen is on
+
+Tracked-On: OAM-97461
```
Comment (basilchew) [inline bsp_diff/caas/device/intel/civ/host/vm-manager/0022-Inhibit-the-Host-suspend-activity.patch:11]: OAM-97461 looks to be from Android 11.
Why is it that this is not part of Android 13 horizontal and we need to port here?
Is there a Jira ticket submitted for horizontal?

File: bsp_diff/caas/device/intel/civ/host/vm-manager/0024-Script-changes-for-wifi-and-BT-firmware-update.patch [hunk 3]
```diff
@@ -0,0 +1,31 @@
+From 4a1d22138852a0ff5799ebe98accf8c8a0973a99 Mon Sep 17 00:00:00 2001
+From: Aiswarya Cyriac <aiswarya.cyriac@intel.com>
+Date: Thu, 6 Jan 2022 12:37:33 +0530
+Subject: [PATCH 2/3] Script changes for wifi and BT firmware update
+
+Tracked-On: OAM-100553
```
Comment (basilchew) [inline bsp_diff/caas/device/intel/civ/host/vm-manager/0024-Script-changes-for-wifi-and-BT-firmware-update.patch:6]: OAM-100553 looks to be from Android 12.
Why do we need this particular firmware?
Why is it that this is not part of Android 13 horizontal and we need to port here?
Is there a Jira ticket submitted for horizontal?

File: bsp_diff/caas/device/intel/civ/host/vm-manager/0025-Add-some-scripts.patch [hunk 4]
```diff
@@ -0,0 +1,183 @@
+From ea792cb12ed2c27760e64b875119cf7ecc305ba0 Mon Sep 17 00:00:00 2001
+From: "Qi, Yadong" <yadong.qi@intel.com>
+Date: Mon, 27 Jul 2020 13:23:46 +0800
+Subject: [PATCH 3/3] Add some scripts
+
+Add some scripts
+
+Tracked-On: OAM-91886
```
Comment (basilchew) [inline bsp_diff/caas/device/intel/civ/host/vm-manager/0025-Add-some-scripts.patch:8]: 1) OAM-91886 is a very old ticket.
It doesn't match the changes here. Please check

2) Why do we need this patch? Why is it not in horizontal?

File: bsp_diff/caas/device/intel/civ/host/vm-manager/0026-Update-civ-1.ini-file-as-per-setup_civ.patch [hunk 5]
```diff
@@ -0,0 +1,129 @@
+From 3b154924967765cc620361586dbf3865a79d269f Mon Sep 17 00:00:00 2001
+From: "Kothapeta, BikshapathiX" <bikshapathix.kothapeta@intel.com>
+Date: Mon, 17 Apr 2023 19:20:08 +0530
+Subject: [PATCH] Update civ-1.ini file as per setup_civ
+
+adb port net attributes are missed as setup-civ-ini.sh script updating.
```
Comment (basilchew) [inline bsp_diff/caas/device/intel/civ/host/vm-manager/0026-Update-civ-1.ini-file-as-per-setup_civ.patch:6]: Any idea why this sample ini file is different from Android 12?
Can we get them to update the horizontal?

File: bsp_diff/caas/vendor/intel/utils/0006-Fix-setup_host-issues-and-updated-working.patch
Comment (basilchew) [inline bsp_diff/caas/vendor/intel/utils/0006-Fix-setup_host-issues-and-updated-working.patch:1]: There are a lot of changes here, not just QEMU patches.
Please check

> host/kernel/linux-intel-lts2021/build_weekly.sh                                                               |    37 -
>  host/kernel/lts2019-chromium/0001-BACKPORT-v5.4-KVM-VMX-Consume-pending-LAPIC-INIT-event-when-exit-o.patch    |    49 +
>  host/kernel/lts2019-chromium/0002-BACKPORT-v5.4-KVM-x86-emulate-wait-for-SIPI-and-SIPI-VMExit.patch           |   186 +
>  host/kernel/lts2019-chromium/0003-v1-vfio-pci-Add-support-for-opregion-v2.0.patch                             |    97 +
>  .../0005-Fix-device-hang-on-TGL-RVP.patch => lts2019-chromium/0004-Fix-device-hang-on-TGL-RVP.patch}          |    18 +-
>  host/kernel/lts2019-chromium/README                                                                           |    33 +
>  host/kernel/lts2019-chromium/build.sh                                                                         |    31 +
>  host/kernel/lts2019-chromium/build_weekly.sh                                                                  |    31 +
>  host/kernel/lts2019-chromium/x86_64_defconfig                                                                 | 10444 ++++++++++++++++++++++++++++++++++++++++++++++++++
>  host/kernel/lts2019-yocto/0001-BACKPORT-v5.4-KVM-VMX-Consume-pending-LAPIC-INIT-event-when-exit-o.patch       |    49 +
>  host/kernel/lts2019-yocto/0002-BACKPORT-v5.4-KVM-x86-emulate-wait-for-SIPI-and-SIPI-VMExit.patch              |   186 +
>  host/kernel/lts2019-yocto/README                                                                              |    33 +
>  host/kernel/lts2019-yocto/build.sh                                                                            |    31 +
>  host/kernel/lts2019-yocto/build_weekly.sh                                                                     |    31 +
>  host/kernel/lts2019-yocto/x86_64_defconfig                                                                    | 10423 ++++++++++++++++++++++++++++++++++++++++++++++++++
>  .../{lts2021-chromium => lts2020-chromium}/0001-Revert-FROMLIST-KVM-mmu-remove-over-aggressive-warni.patch    |    12 +-
>  host/kernel/lts2020-chromium/0002-Revert-BACKPORT-FROMLIST-KVM-arm64-mmu-use-gfn_to_pf.patch                  |    97 +
>  .../0003-Revert-BACKPORT-FROMLIST-KVM-x86-mmu-use-gfn_to_pfn_.patch}                                          |   133 +-
>  .../{lts2021-chromium => lts2020-chromium}/0004-Revert-FROMLIST-KVM-mmu-introduce-new-gfn_to_pfn_pag.patch    |    34 +-
>  host/kernel/lts2020-chromium/0005-KVM-x86-emulate-wait-for-SIPI-and-SIPI-VMExit.patch                         |   184 +
>  host/kernel/lts2020-chromium/0006-Fix-device-hang-on-TGL-RVP.patch                                            |    77 +
>  host/kernel/lts2020-chromium/0007-Revert-UPSTREAM-drm-i915-gvt-Introduce-per-object-lo.patch                  |   129 +
>  .../0008-Update-header-config-to-allow-older-fw-version.patch}                                                |     8 +-
>  host/kernel/lts2020-chromium/0009-ISH-driver-Support-for-AlderLake.patch                                      |    45 +
>  host/kernel/{lts2021-chromium => lts2020-chromium}/README                                                     |     2 +-
>  host/kernel/{lts2021-chromium => lts2020-chromium}/build.sh                                                   |    13 +-
>  host/kernel/{lts2021-chromium => lts2020-chromium}/build_weekly.sh                                            |    17 +-
>  host/kernel/{lts2021-chromium => lts2020-chromium}/x86_64_defconfig                                           |     2 +-
>  host/kernel/lts2020-yocto/0001-v1-vfio-pci-Add-support-for-opregion-v2.0.patch                                |    97 +
>  host/kernel/{linux-intel-lts2021 => lts2020-yocto}/README                                                     |     2 +-
>  host/kernel/lts2020-yocto/build.sh                                                                            |    33 +
>  host/kernel/lts2020-yocto/build_weekly.sh                                                                     |    31 +
>  host/kernel/{linux-intel-lts2021 => lts2020-yocto}/x86_64_defconfig                                           |  1293 +++----
>  host/kernel/lts2021-chromium/0002-Revert-FROMLIST-KVM-arm64-mmu-use-gfn_to_pfn_page.patch                     |   104 -
>  host/ovmf/0002-Fix-vla-parameter-error.patch                                                                  |    52 -
>  host/qemu/0004-Add-the-rgba-rgbx-support.patch                                                                |    55 -
>  host/qemu/0005-ui-gtk-new-param-monitor-to-specify-target-monitor-f.patch                                     |    96 -
>  host/qemu/0006-Adapt-APP-landscape-or-portrait-mode-display-dynamic.patch                                     |   236 --
>  host/qemu/0008-Fix-drawing-delay-issue-in-cases-of-resize-and-rotat.patch                                     |    36 -
>  .../kernel/lts2019-chromium/0001-BACKPORT-v5.4-KVM-VMX-Consume-pending-LAPIC-INIT-event-when-exit-o.patch     |    49 +
>  .../patches/kernel/lts2019-chromium/0002-BACKPORT-v5.4-KVM-x86-emulate-wait-for-SIPI-and-SIPI-VMExit.patch    |   186 +
>  host/qemu/patches/kernel/lts2019-chromium/0003-v1-vfio-pci-Add-support-for-opregion-v2.0.patch                |    97 +
>  host/qemu/patches/kernel/lts2019-chromium/0004-Fix-device-hang-on-TGL-RVP.patch                               |    77 +
>  host/qemu/patches/kernel/lts2019-chromium/README                                                              |    33 +
>  host/qemu/patches/kernel/lts2019-chromium/build.sh                                                            |    31 +
>  host/qemu/patches/kernel/lts2019-chromium/build_weekly.sh                                                     |    31 +
>  host/qemu/patches/kernel/lts2019-chromium/x86_64_defconfig                                                    | 10444 ++++++++++++++++++++++++++++++++++++++++++++++++++
>  .../kernel/lts2019-yocto/0001-BACKPORT-v5.4-KVM-VMX-Consume-pending-LAPIC-INIT-event-when-exit-o.patch        |    49 +
>  host/qemu/patches/kernel/lts2019-yocto/0002-BACKPORT-v5.4-KVM-x86-emulate-wait-for-SIPI-and-SIPI-VMExit.patch |   186 +
>  host/qemu/patches/kernel/lts2019-yocto/README                                                                 |    33 +
>  host/qemu/patches/kernel/lts2019-yocto/build.sh                                                               |    31 +
>  host/qemu/patches/kernel/lts2019-yocto/build_weekly.sh                                                        |    31 +
>  host/qemu/patches/kernel/lts2019-yocto/x86_64_defconfig                                                       | 10423 ++++++++++++++++++++++++++++++++++++++++++++++++++
>  host/qemu/patches/kernel/lts2020-chromium/0001-Revert-FROMLIST-KVM-mmu-remove-over-aggressive-warni.patch     |    48 +
>  host/qemu/patches/kernel/lts2020-chromium/0002-Revert-BACKPORT-FROMLIST-KVM-arm64-mmu-use-gfn_to_pf.patch     |    97 +
>  host/qemu/patches/kernel/lts2020-chromium/0003-Revert-BACKPORT-FROMLIST-KVM-x86-mmu-use-gfn_to_pfn_.patch     |   329 ++
>  host/qemu/patches/kernel/lts2020-chromium/0004-Revert-FROMLIST-KVM-mmu-introduce-new-gfn_to_pfn_pag.patch     |   364 ++
>  host/qemu/patches/kernel/lts2020-chromium/0005-KVM-x86-emulate-wait-for-SIPI-and-SIPI-VMExit.patch            |   184 +
>  host/qemu/patches/kernel/lts2020-chromium/0006-Fix-device-hang-on-TGL-RVP.patch                               |    77 +
>  host/qemu/patches/kernel/lts2020-chromium/0007-Revert-UPSTREAM-drm-i915-gvt-Introduce-per-object-lo.patch     |   129 +
>  host/qemu/patches/kernel/lts2020-chromium/0008-Update-header-config-to-allow-older-fw-version.patch           |    29 +
>  host/qemu/patches/kernel/lts2020-chromium/0009-ISH-driver-Support-for-AlderLake.patch                         |    45 +
>  host/qemu/patches/kernel/lts2020-chromium/README                                                              |    37 +
>  host/qemu/patches/kernel/lts2020-chromium/build.sh                                                            |    33 +
>  host/{kernel/linux-intel-lts2021/build.sh => qemu/patches/kernel/lts2020-chromium/build_weekly.sh}            |    15 +-
>  host/qemu/patches/kernel/lts2020-chromium/x86_64_defconfig                                                    | 10780 +++++++++++++++++++++++++++++++++++++++++++++++++++
>  host/qemu/patches/kernel/lts2020-yocto/0001-v1-vfio-pci-Add-support-for-opregion-v2.0.patch                   |    97 +
>  host/qemu/patches/kernel/lts2020-yocto/README                                                                 |    37 +
>  host/qemu/patches/kernel/lts2020-yocto/build.sh                                                               |    33 +
>  host/qemu/patches/kernel/lts2020-yocto/build_weekly.sh                                                        |    31 +
>  host/qemu/patches/kernel/lts2020-yocto/x86_64_defconfig                                                       | 11027 +++++++++++++++++++++++++++++++++++++++++++++++++++++
>  host/qemu/patches/lg/0001-UPSTREAM-common-fix-build-for-newer-versions-of-binu.patch                          |    51 +
>  host/qemu/patches/lg/0002-INTERNAL-Don-t-close-LG-client-when-spice-is-disconn.patch                          |    64 +
>  host/qemu/patches/lg/0003-INTERNAL-Fix-spice-fouce-issue.patch                                                |   119 +
>  host/qemu/patches/lg/0004-INTERNAL-Add-support-for-dynamic-window-name.patch                                  |    24 +
>  host/qemu/patches/lg/0005-INTERNAL-Flag-LG-client-is-ready-to-receive-frame.patch                             |    26 +
>  host/qemu/patches/lg/0006-Add-support-for-virtual-input.patch                                                 |   249 ++
>  host/qemu/patches/lg/0007-INTERNAL-Remove-debug-info-log.patch                                                |    28 +
>  host/qemu/patches/lg/0008-When-frame-is-rotated-rotate-it-in-shader.patch                                     |   223 ++
>  host/qemu/patches/lg/0009-Exchange-Width-and-Height-when-rotating.patch                                       |    94 +
>  host/qemu/patches/lg/0010-INTERNAL-Improve-LG-Client-performace.patch                                         |    91 +
>  host/qemu/patches/lg/0011-Add-clipboard-support-over-vsock.patch                                              |   675 ++++
>  host/qemu/patches/lg/0012-Handle-XError-gracefully-to-avoid-crash.patch                                       |   191 +
>  host/qemu/patches/lg/0013-Add-input-support-in-Landscape-from-Rotate-operation.patch                          |   579 +++
>  host/qemu/patches/lg/0014-Revert-Add-input-support-in-Landscape-from-Rotate-op.patch                          |   451 +++
>  host/qemu/patches/lg/0015-Fixed-the-looking-glass-client-rendering-messed-issu.patch                          |    52 +
>  host/qemu/patches/lg/0016-Add-Fullscreen-input-support-for-looking-glass.patch                                |   175 +
>  host/qemu/patches/lg/0016-Fixed-the-app-window-not-refreshed-issue-after-resum.patch                          |    36 +
>  host/qemu/patches/lg/0017-Fixed-the-render-thread-frame-thread-race-codition.patch                            |    64 +
>  host/qemu/patches/ovmf/0001-OvmfPkg-add-IgdAssignmentDxe.patch                                                |   757 ++++
>  host/qemu/{ => patches/qemu}/0001-Revert-Revert-vfio-pci-quirks.c-Disable-stolen-memor.patch                  |     0
>  host/qemu/{ => patches/qemu}/0002-qemu-change-fence-poll-time-by-current-workload.patch                       |     0
>  host/qemu/{ => patches/qemu}/0003-Disable-EDID-auto-generation-in-QEMU.patch                                  |     0
>  host/qemu/patches/qemu/0031-Set-the-fourcc-if-it-is-0.patch                                                   |    30 +
>  .../qemu/0033-CiV-SRIOV-virtio-gpu-power-management-qemu.patch}                                               |    16 +-
>  host/qemu/patches/qemu/0034-Fix-virtio2d-color-r-b-reversed-issue.patch                                       |    41 +
>  96 files changed, 72067 insertions(+), 1529 deletions(-)
>

**Review by basilchew — CHANGES_REQUESTED** [review]

File: bsp_diff/caas/device/intel/civ/host/vm-manager/0026-Update-civ-1.ini-file-as-per-setup_civ.patch (hunk 5 above)
Comment (bkothapx) [reply bsp_diff/caas/device/intel/civ/host/vm-manager/0026-Update-civ-1.ini-file-as-per-setup_civ.patch:6]: Its already similar to Horizontal as rebased from Horizontal and setup_civ_ini.sh file script is updating based on this sample file, adb emulation ports are not  added in civ-sriov.ini file for this reason I have to made this change as per the Prashanth Suggestion.

File: bsp_diff/caas/vendor/intel/utils/0006-Fix-setup_host-issues-and-updated-working.patch
Comment (bkothapx) [reply bsp_diff/caas/vendor/intel/utils/0006-Fix-setup_host-issues-and-updated-working.patch:1]: Yes, vendor/intel/utils which is updated from Horizontal during setup_host.sh script execution its failing during **qemu patch apply hence updated to horizontal working patches which resolved this issue.**

**Pull request closed without merging**